pkg/kubectl/cmd/create_configmap.go

```diff
@@ -55,7 +60,7 @@ var (
 // ConfigMap is a command to ease creating ConfigMaps.
 func NewCmdCreateConfigMap(f *cmdutil.Factory, cmdOut io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "configmap NAME [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run]",
+		Use:     "configmap NAME [--from-file=[key=]source] [--from-binary-file=[key=]source] [--from-literal=key1=value1] [--dry-run]",
```

Comment:
I don't agree. It should be very easy to read the file and test whether it is a valid utf-8 string, and includes no nulls or control characters. Anything this misclassifies also shouldn't matter.

And logically the two fields are the same, since their keys must not overlap.